bugbug/models/bugtype.py

```diff
@@ -103,7 +33,19 @@ def __init__(self, lemmatization=False, historical=False):
             bug_features.Severity(),
             # Ignore keywords that would make the ML completely skewed
             # (we are going to use them as 100% rules in the evaluation phase).
-            bug_features.Keywords(set(KEYWORD_DICT.keys())),
+            bug_features.Keywords(
+                {
+                    "sec-",
+                    "csectype-",
+                    "memory-",
+                    "crash",
+                    "crashreportid",
+                    "perf",
+                    "topperf",
+                    "main-thread-io",
+                    "power",
+                }
```

Comment:
I feel this is a bit fragile and could be an error-prone. If any of the functions add a new keyword, it will be highly likely to miss adding it here.

One solution would be to convert the functions to feature classes. Then, the keywords will be stored as attributes from these classes:

```py
self.label_extractors = [
    IsPerformanceBug(),
    IsMemoryBug(),
    IsPowerBug(),
    IsSecurityBug(),
    IsCrashBug(),
]
```

We could extract the keywords by iterating over the label extractors:

```py
keywords = {
    keyword
    for extractor in self.label_extractors
    for keyword in extractor.keywords
}
```

Reply:
Updated:

https://github.com/mozilla/bugbug/blob/906aa2bb12944ffa1f02a7837cb9f3fb0b9cc46a/bugbug/models/bugtype.py#L30-L51

Reply:
Nice! Thank you.